CHANGELOG.md

```diff
@@ -22,6 +22,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * Show paths to config files when configuration errors occur
 
+* Added optional depth argument to `descendants`, using existing functions;
```

Comment:
What do you mean by "using existing functions" here? If it's about the implementation, I think it should be removed from here because users won't care

Reply:
Yeah, just acknowledging that the library functionality already existed but wasn't exposed. I can remove that bit.